README.md

```diff
@@ -787,8 +785,8 @@ unsplash.collections.listRelatedCollections(88)
 
 <div id="search" />
 
-### search.all(keyword, page, per_page)
```

Comment:
Removing documentation for this because it isn't something that will necessarily be indefinitely supported and it currently isn't in use by anyone at scale other than us.